src/Groups/GAPGroups.jl

```diff
@@ -2269,7 +2269,7 @@ nonzero then `pairs` is the vector of syllables of `x`, see [`syllables`](@ref).
 julia> G = free_group(2);  pairs = [1 => 3, 2 => -1];
 
 julia> x = G(pairs)
-f1^3*f2^-1
+x1^3*x2^-1
```

Comment:
With you default char hack below, this shouldn't be changed

Reply:
```suggestion
f1^3*f2^-1
```